pkg/apb/types.go

```diff
@@ -200,3 +201,10 @@ type RecoverStatus struct {
 	InstanceID uuid.UUID `json:"id"`
 	State      JobState  `json:"state"`
 }
+
+// ExecutionContext - Contains the information necessary to track and clean up an APB run
+type ExecutionContext struct {
```

Comment:
Why couldn't we reuse the `Context` struct which already has a `Namespace` and `Platform`?